python/Scripts/nodedef_documentation.md

```diff
@@ -2,47 +2,47 @@
 - *Type*: surfaceshader
 - *Doc*: Autodesk standard surface shader
 
-| Name | Type | Default Value | UI min | UI max | UI group | Description | UI Advanced | Connectable |
```

Comment:
I think this got checked in by mistake :). Can you just delete the file as part of the check-in. Thanks.